### ISSUE TITLE
1.3+ fails to build

### ISSUE THREAD

**Issue by sententia** [revolunet/sublimetext-markdown-preview#310]
Hi, 

Been working fine until I got version 1.3+ 

I go to preview in the browser and I get that it building the markdown but then I get this error in the console

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 550, in run_
    return self.run(edit, **args)
  File "MarkdownPreview in /Users/rhystague/Library/Application Support/Sublime Text 3/Installed Packages/Markdown Preview.sublime-package", line 1173, in run
  File "MarkdownPreview in /Users/rhystague/Library/Application Support/Sublime Text 3/Installed Packages/Markdown Preview.sublime-package", line 763, in run
  File "MarkdownPreview in /Users/rhystague/Library/Application Support/Sublime Text 3/Installed Packages/Markdown Preview.sublime-package", line 704, in convert_markdown
  File "MarkdownPreview in /Users/rhystague/Library/Application Support/Sublime Text 3/Installed Packages/Markdown Preview.sublime-package", line 1100, in parser_specific_convert
  File "/Users/rhystague/Library/Application Support/Sublime Text 3/Packages/Markdown Preview/markdown/__init__.py", line 384, in convert
  File "markdown.extensions.headeranchor in /Users/rhystague/Library/Application Support/Sublime Text 3/Installed Packages/Markdown Preview.sublime-package", line 123, in run
  File "markdown.extensions.toc in /Users/rhystague/Library/Application Support/Sublime Text 3/Installed Packages/Markdown Preview.sublime-package", line 28, in slugify
TypeError: must be str, not builtin_function_or_method
```

I've had to revert to version 1.2 for it to work again.

**Comment by facelessuser** [comment]
I am going to need more info. It looks like you are on OSX with ST3?  I am also on OSX with ST3 except everything is working for me.  

- Did you try restarting Sublime after installing?
- What do your settings look like?
- Do you have example markdown to test?

**Comment by sententia** [comment]
Yes that is right, ST3, and OSX.  Restarted after installing, then did manual install, and then installed through folder. There are my settings:

```
{
   
    "enable_highlight": true,
    "enabled_extensions": ["extra", "github", "codehilite(guess_lang=False,pygments_style=github)"],
    "enabled_parsers": ["markdown"]
}
```
When selecting markdown to preview in browser it comes up as converting markdown with Python markdown... and then goes away and then  nothing.

**Comment by facelessuser** [comment]
Great, your settings allowed me to reproduce.  I'll take a look at what's going on.

**Comment by facelessuser** [comment]
Found the issue.

**Comment by facelessuser** [comment]
Fix in [`1.3.2`](https://github.com/revolunet/sublimetext-markdown-preview/releases/tag/1.3.2)
